lib/Paws/Net/RetryCallerRole.pm

```diff
@@ -15,8 +15,7 @@ package Paws::Net::RetryCallerRole;
     do {
       $tracker->one_more_try;
 
-      my ($status, $content, $headers) = $self->send_request($service, $call_object);
-      my $result = $self->caller_to_response($service, $call_object, $status, $content, $headers);
+      my $result = $self->send_request($service, $call_object);
```

Comment:
This should be the two lines from the left of the diff (probably the cause for test failures)